zadani/soft/style.css

```diff
@@ -15,29 +16,311 @@ body {
     font-family: 'Merriweather', serif;
     font-size: 16px;
     line-height: 1.6;
+    
 }
 
 
 /* HLAVICKA */
+.hlavicka {    
+    background-image: linear-gradient(rgba(0,0,0,0.3),rgba(0,0,0,0.3)), 
+                    url("../../images/island-hlavicka.jpg"); 
+    background-position: center;
+    background-size: cover;
+    min-height: 200px;
+    text-align: center;
+    padding: 5px;
+    margin: 0
+}
 
 .titulek {
-    font-family: 'Raleway', serif;
+    font-size: 50px;
+    font-family:'Raleway', serif;
+    color:white;
+    padding:20px;
+    margin:0px;
+}
+
+.perex {
+    font-size: 12px;
+    color:white;
+    padding:20px;
+    margin:0px;
+}
+
+.autor {
+    color: rgba(236, 233, 233, 0.836);
+    font-size: 12px;
+    font-style: italic;
+    padding:30px 20px 30px 20px;
+    position: relative;
 }
 
+.autor::after {
```

Comment:
Pěkné použití pseudoelementu 👍 
Stejně tak by šel i before (možná by byl lehoulince vhodnější, protože čára je "před" autorem)